assets/js/script.js

```diff
@@ -145,7 +145,7 @@ function addSheets(){
         for (var j=0 ; j < 3 ; j++){
             var span = document.createElement('span');
             var type = ['survey', 'choices', 'settings'][j];
-            if (j > 0) {
+            if (j > 0 && tds[j]) {
```

Comment:
I don't see the addSheets exception (not even sure where to look), but I trust you.